### ISSUE TITLE
ng-model directive is not updating the host data in Dart

### ISSUE THREAD

**Issue by luisvt** [angular/angular#4646]
I have next component:

```Dart

@Component(selector: 'my-app')
@View(
    directives: const [FORM_DIRECTIVES],
    template: '''
Query text: {{searchQuery}}<br>
<input type='text' [(ng-model)]="searchQuery">
<input type="button" value="Change Me!" (click)="onChangeMeClicked()">
''')
class AppComponent {
  String searchQuery = 'hello';

  onChangeMeClicked() {
    print(searchQuery); // prints old value
    searchQuery = 'change me';
    print(searchQuery); // prints 'change me' - new value
  }
}
```

but the input-text is not being updated when I click on the input-button. After clicking in the console appears correctly the changes to variable `searchQuery`, but the input-text is not updated.

Also the text on `Query text: {{searchQuery}}` is updated correctly in both cases, after clicking input button and after editing input-text.

**Comment by yjbanov** [comment]
@vsavkin feel free to drop priority if there's a workaround.

**Comment by luisvt** [comment]
It looks resolved for version 2.0.0-alpha.44
